### PR TITLE
protocol_validate_fscq: output adapted to standard attributes viewer

### PR DESCRIPTION
**Pull request by DaniDelHoyo**
FSC-Q protocol output is AtomStruct.
Input AtomStruct as object

### CODE REVIEW

**Comment by DaniDelHoyo** [comment]
Los valores por defecto para el rango de fscq he puesto (-3,3), no se si habria que cambiarlo

**Review by erneyramirez — APPROVED** [review]

**Review by DStrelak — CHANGES_REQUESTED** [review]
PR name does not match the content (this is clearly FSCQ related PR)

**Comment by DaniDelHoyo** [comment]
> PR name does not match the content (this is clearly FSCQ related PR) 
Hi David, the PR adapts the viewer of the FSC-Q protocol to the general attributes viewer I developed, that's why the name.
If you think the name still must be changed, we'll do it

> Also please add tests for the FCSQ protocol. At least I don't see any at the buildbot.
do you have any test for the protocol @erneyramirez ?

**Comment by erneyramirez** [comment]
@DStrelak I have to do the test, I had it pending. Pass the PR so Dani can get rid of it and I'll take care of the test

**Comment by DStrelak** [comment]
> he PR adapts the viewer of the FSC-Q protocol to the general attributes viewer

In that case yes, please change the name of the PR (convention is roughly protocol_name: work_done)

**Comment by DStrelak** [comment]
> @DStrelak I have to do the test, I had it pending. Pass the PR so Dani can get rid of it and I'll take care of the test

Please accompany this PR with the test directly. It's to avoid a situation in which we have not working code already in the devel.
Alternatively we can put this PR on hold, and you can create the test in a derived branch, eventually merging it to this PR and then to devel.

**Comment by DaniDelHoyo** [comment]
There you have it, the test and the name is changed

File: xmipp3/protocols/protocol_validate_fscq.py [hunk 1]
```diff
@@ -319,58 +335,58 @@ def runBlocresStep(self, i):
                         env=bsoft.Plugin.getEnviron(bsoftVersion=bsoft.V1_9_0))
```
Comment (DStrelak) [inline xmipp3/protocols/protocol_validate_fscq.py:335]: the version of the program should be defined in a single place

Comment (DStrelak) [inline xmipp3/protocols/protocol_validate_fscq.py:335]: I also get `Protocol failed: module 'bsoft' has no attribute 'V1_9_0'`

**Review by DStrelak — CHANGES_REQUESTED** [review]
Test fails with
```
Error: Option -criterio is not defined!
Option errors: 1
Error: Option -criterio is not defined!
Option errors: 1
Traceback (most recent call last):
```

**Review by DStrelak — APPROVED** [review]

**Pull request merged**